pkg/controller/kieapp/defaults/defaults_test.go

```diff
@@ -5951,3 +5951,64 @@ func TestCRServerCPULimitAndRequestUsingMilicores(t *testing.T) {
 	assert.Equal(t, values.Requests.Cpu(), env.Servers[0].DeploymentConfigs[0].Spec.Template.Spec.Containers[0].Resources.Requests.Cpu())
 	assert.Equal(t, values.Limits.Cpu(), env.Servers[0].DeploymentConfigs[0].Spec.Template.Spec.Containers[0].Resources.Limits.Cpu())
 }
+
+func TestRhdmEnvironmentWithKafkaJBPM(t *testing.T) {
```

Comment:
RHDM environment with Kafka JBPM configuration does not make sense, as RHDM Kie Server do not have enabled JBPM capabilities. For RHDM it would be better to not have this configuration (but this I'm worried is not possible to have in KieApp) or have parameter KIE_SERVER_KAFKA_EXT_ENABLED override to false with some exception.
Please tests this configuration on RHPAM environment.

Reply:
+1

Reply:
ok